presto-accumulo/pom.xml

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.152-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-accumulo</artifactId>
+    <description>Presto - Accumulo Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <accumulo.version>1.6.2</accumulo.version>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.test.fork-mode>always</air.test.fork-mode>
```

Comment:
My memory is a bit fuzzy on this... I believe the issue here is that the test suites that use the `DistributedQueryRunner` try to create the TPC-H tables under the `accumulo` catalog again, causing the build to fail.  Additionally, when one suite shuts down, it deletes the tables while the other is still running.   I think there was a third issue of the shutdown hook kicking in when one suite finished, dismantling the MiniAccumuloCluster while other tests are still using it. I spent some time trying to resolve this issue it but ultimately gave up and set this property to press onward.  I'll give it another shot.